common/threadloop.hpp

```diff
@@ -17,13 +19,19 @@ namespace ILLIXR {
  */
 class threadloop : public plugin {
 public:
+	threadloop(std::string name_, phonebook* pb_) : plugin(name_, pb_) { }
+
 	/**
 	 * @brief Starts the thread.
 	 */
 	void start() override {
 		_m_thread = std::thread([this]() {
-		while (!should_terminate()) {
-				_p_one_iteration();
+			while (!should_terminate()) {
+				std::chrono::nanoseconds duration;
+				{   TIME_THIS_BLOCK(duration);
+					_p_one_iteration();
+				}
+				std::cout << "cpu_timer," << get_name() << "," << duration.count() << std::endl;
```

Comment:
I'm assuming the use of `cout` is intentional? That is, we dump to `cout` and then use `grep` to get what we need?

Reply:
That would make sense to me. #UnixPhilosophy

Reply:
Exactly, `make run.dbg grep '^cpu_timer,' > all_timers.csv`.

Reply:
If I logged to a file, that file object would need to be globally accessible at every site I need to write timing info. Currently, this is the only site, but that will change.